compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +66,28 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getEnableLoopPeelingAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, kLoopPeelingAttrName);
+}
+
+bool isLoopPeelingEnabled(FunctionOpInterface funcOp) {
+  auto peelAttrName = getEnableLoopPeelingAttrName(funcOp->getContext());
+  auto trueAttr = BoolAttr::get(funcOp->getContext(), true);
```

Comment:
Do we use `UnitAttr` instead? The attribute name is `enable_loop_peeling`. It is clear that we want peeling when the attribute is present. What do you think?

Reply:
+1 to a unit attribute (